app/views/spree/admin/webhooks_subscribers/show.html.erb

```diff
@@ -39,12 +39,12 @@
       <tbody>
       <% @events.each do |event| %>
         <tr id="<%= spree_dom_id event %>">
-          <td><%= event.name %></td>
+          <td><%= event.name.html_safe %></td>
           <td><%= active_badge(event.success) %></td>
-          <td><%= event.created_at %></td>
-          <td><%= event.response_code %></td>
-          <td><%= event.execution_time %></td>
-          <td><%= event.request_errors.gsub('[SPREE WEBHOOKS]', '') %></td>
+          <td><%= event.created_at.html_safe %></td>
```

Comment:
@aleksandarpetrushev that's the opposite - https://apidock.com/rails/String/html_safe - it will return the string as it is, we don't want raw data